src/Compiler/Facilities/Activity.fs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
```

Comment:
Looks to me like this should be in Utilities.  Basically 

* Utilities = General Utilities
* Facilities = Compiler-specific Utilities 

I don't mind if we rename these

* Utilities --> GeneralUtilities
* Facilities --> CompilerUtilities

This looks like a General utility - it would have use outside the compiler and is conceptually and implementation-wise independent of it.

BTW what's the `open Internal.Utilities.Library` used here for?

Reply:
I've done the move.

Also removed the open as it was unused.